contracts/Dogshit.vy

```diff
@@ -39,7 +39,7 @@ nonces: public(HashMap[address, uint256])
 DOMAIN_SEPARATOR: public(bytes32)
 DOMAIN_TYPE_HASH: constant(bytes32) = keccak256('EIP712Domain(string name,string version,uint256 chainId,address verifyingContract)')
 PERMIT_TYPE_HASH: constant(bytes32) = keccak256("Permit(address owner,address spender,uint256 value,uint256 nonce,uint256 deadline)")
-wrapToken: constant(address) = 0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c
+wrapToken: constant(address) = 0x1f7216fdb338247512ec99715587bb97bbf96eae
```

Comment:
This needs to be a checksum address (mix of upper and lower case).  You can search the contract on bsc scan and then copy the checksum address from where it is displayed.